src/runtime.js

```diff
@@ -87,6 +88,7 @@ export function installRuntimeServices(global) {
   installViewerService(global);
   installViewportService(global);
   installHistoryService(global);
+  installStorageService(global);
```

Comment:
Actual instantiation is lazy anyway.

On Tue, Sep 13, 2016 at 4:42 PM, Dima Voytenko <notifications@github.com>
wrote:

> In src/runtime.js
> <https://github.com/ampproject/amphtml/pull/4944#discussion_r78665110>:
>
> > @@ -87,6 +88,7 @@ export function installRuntimeServices(global) {
> >    installViewerService(global);
> >    installViewportService(global);
> >    installHistoryService(global);
> > +  installStorageService(global);
>
> @cramforce <https://github.com/cramforce> should we push promisable
> services to be installed in a micro-task here so that we don't block the
> initial render?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/ampproject/amphtml/pull/4944/files/38ae18a72ba2f5a6a8e5dbfb212ee19888c057fa#r78665110>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAFeT-DA__E8m925fhVJtGs_ptQq6m64ks5qpzTLgaJpZM4J7GGg>
> .
>